packages/web/src/pages/match/single-match.tsx

```diff
@@ -93,6 +94,12 @@ const SingleMatch: React.FC = () => {
       </div>
     );
   } else {
+
+    let startgamedate = new Date(matchData.startgametime * 1000); 
+    let currentday = new Date();
+    let days = config.matchAreStoredForDays - differenceInDays(currentday, startgamedate);
+    let expiryDate = new Date(new Date().setDate(currentday.getDate() + days)).toLocaleDateString('en-us', { weekday:"long", year:"numeric", month:"short", day:"numeric"});
```

Comment:
I think that all these variables can be `const`. It's better to use it https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

We can let this to be empty. 
```
toLocaleDateString('en-us', { weekday:"long", year:"numeric", month:"short", day:"numeric"})
```
We are using dates on more parts of the app and it's always formatted for the user - depending on where they live :)

Reply:
Thanks @petrvecera 
I will update them now. 
Thanks :)

Reply:
@petrvecera  done. please review it now